x/execution/internal/keeper/params.go

```diff
@@ -0,0 +1,34 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/mesg-foundation/engine/x/execution/internal/types"
+)
+
+// Default parameter namespace
+const (
+	DefaultParamspace = types.ModuleName
+)
+
+// MinPrice - Minimum price of an execution
+func (k Keeper) MinPrice(ctx sdk.Context) sdk.Coins {
+	var coinstr string
+	k.paramstore.Get(ctx, types.KeyMinPrice, &coinstr)
+	price, err := sdk.ParseCoins(coinstr)
+	if err != nil {
+		panic(err)
```

Comment:
can't you return the error here?

Reply:
I delegated the parsing to the calling function